### ISSUE TITLE
libvorbis.module.min.js.mem file can't be loaded

### ISSUE THREAD

**Issue by d668** [sb2702/audioRecord.js#1]
IIS  (and others) don't allow mem files, can you change format?

**Comment by sb2702** [comment]
Sure - Looking into it right now

**Comment by Rillke** [comment]
Maybe I am on the wrong path; according to MSDN, [there is configuration for MIME types](https://msdn.microsoft.com/en-us/library/bb742440.aspx).

Don't make it `js`, otherwise [CloudFlare will possibly corrupt the files](https://github.com/Rillke/opusenc.js/issues/3) ;-)

I guess something that suggests it could be a binary file like `zip` or `lzh` could work for most servers. It is possibly important not to choose an image format because some providers will try to scale them for mobile etc....

**Comment by sb2702** [comment]
Well, so the original problem I ran into was finding a javascript library that could do the OGG Vorbis encoding. I found this library: https://github.com/itsjoesullivan/libvorbis.js/tree/master for compiling native libraries into javascript (with emscripten), but it's been giving me trouble on my system.   I can't remember where exactly I found the minified, optimized version of libvorbis.module.min.js, but I found it somewhere, found out that it worked, and integrated it into the project. 

To fix this, I just need a compiled version of libvorbis https://github.com/itsjoesullivan/libvorbis.js/tree/master or something equivalent, that's not optimized - I just found https://github.com/Garciat/libvorbis.js, so I'm checking into it now

As a sidenote- the mem file suggess that this copy of libvorbis is optimized, meaning it'll perform better
